src/components/dashboard/employees/Employees.js

```diff
@@ -30,25 +33,45 @@ class Employees extends Component {
 
   onCreateClick = () => {
     this.props.history.push(routes.createEmployee);
-  }
+  };
 
-  render() {    
+  render() {
     return (
       <div className="employees">
         <div className="employees-container">
-          <TitleBar title="Manage your employees" subtitle="Add a new employee to send them a new hire box" />
-          <div className="contain-actions">
-            <span className="create-button" onClick={this.onCreateClick}>
-              <span className="create-employee-text">Add Employee</span>
-              <MdAddCircle />
-            </span>
-          </div>
-          <div className="employees-table">
-            <Table data={this.props.employees} columns={columns} />
-          </div>
-          <div className="total-row">Total Employees: {this.props.employees.length}</div>
+          <TitleBar
+            title="Manage your employees"
+            subtitle="Add a new employee to send them a new hire box"
+          />
+          {this.props.employees.length <= 0 && (
+            <div className="employees-zero-state">
+              <div className="employees-zs-text">
+                You haven't added any employees to send swag to yet! Start adding employees by clicking
+                on the add button.
```

Comment:
github is stupid and renders this `red` block when an apostrophe isn't escaped in html content. You don't need to fix it just FYI for easier readability of future PR's :)

Reply:
ooh I was wondering why haha